libs/safekeeper_api/src/models.rs

```diff
@@ -50,6 +50,8 @@ pub struct SkTimelineInfo {
     pub safekeeper_connstr: Option<String>,
     #[serde(default)]
     pub http_connstr: Option<String>,
+    #[serde(default = "lsn_invalid")]
+    pub standby_flush_lsn: Lsn,
```

Comment:
This would be a good place for a doc comment explaining the meaning and purpose of the field.

Reply:
Done